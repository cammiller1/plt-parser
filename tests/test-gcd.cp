## find the GCD ##
## https://www.geeksforgeeks.org/gcd-in-python/ ##

## Python code to demonstrate naive ##
## method to compute gcd ( recursion ) ##
  
def int hcfnaive(int a, int b) {
    if (b == 0) {
        return a;
    } else {
        return hcfnaive(b, a % b);
    }
}

print("The gcd of 60 and 48 is");
int result;
result = hcfnaive(60, 48);
print(result);