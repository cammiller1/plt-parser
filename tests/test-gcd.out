The gcd of 60 and 48 is
12